<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>调试工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>1030</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>串口</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>串口设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <widget class="QPushButton" name="pushButton_6">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>140</y>
                <width>75</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>打开串口</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="checkBox_5">
              <property name="geometry">
               <rect>
                <x>330</x>
                <y>100</y>
                <width>71</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>16进制</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="checkBox_7">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>100</y>
                <width>71</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>数据量</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>20</y>
                <width>54</width>
                <height>12</height>
               </rect>
              </property>
              <property name="text">
               <string>  接收</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>60</y>
                <width>54</width>
                <height>12</height>
               </rect>
              </property>
              <property name="text">
               <string>  发送</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="lineEdit_5">
              <property name="geometry">
               <rect>
                <x>330</x>
                <y>20</y>
                <width>71</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLineEdit" name="lineEdit_6">
              <property name="geometry">
               <rect>
                <x>330</x>
                <y>50</y>
                <width>71</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_8">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>140</y>
                <width>75</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_9">
              <property name="geometry">
               <rect>
                <x>330</x>
                <y>140</y>
                <width>75</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>清空</string>
              </property>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>30</y>
                <width>54</width>
                <height>12</height>
               </rect>
              </property>
              <property name="text">
               <string>串口</string>
              </property>
             </widget>
             <widget class="QComboBox" name="comboBox">
              <property name="geometry">
               <rect>
                <x>100</x>
                <y>20</y>
                <width>69</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>70</y>
                <width>54</width>
                <height>12</height>
               </rect>
              </property>
              <property name="text">
               <string>波特率</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>110</y>
                <width>54</width>
                <height>12</height>
               </rect>
              </property>
              <property name="text">
               <string>数据位</string>
              </property>
             </widget>
             <widget class="QComboBox" name="comboBox_2">
              <property name="geometry">
               <rect>
                <x>100</x>
                <y>60</y>
                <width>69</width>
                <height>22</height>
               </rect>
              </property>
              <item>
               <property name="text">
                <string>1200</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2400</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>38400</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>115200</string>
               </property>
              </item>
             </widget>
             <widget class="QComboBox" name="comboBox_3">
              <property name="geometry">
               <rect>
                <x>100</x>
                <y>100</y>
                <width>69</width>
                <height>22</height>
               </rect>
              </property>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
             </widget>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0" colspan="5">
               <widget class="QLineEdit" name="lineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="checkBox">
                <property name="text">
                 <string>16进制</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="checkBox_2">
                <property name="text">
                 <string>定时</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="lineEdit_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>ms/次</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QPushButton" name="pushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>发送</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>UDP</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>TCP</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>断开连接</string>
     </property>
     <addaction name="action_4"/>
     <addaction name="actionudp"/>
     <addaction name="actiontcp"/>
    </widget>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="menu_3"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>串口工具模式</string>
     </property>
     <addaction name="action_9"/>
     <addaction name="action16"/>
    </widget>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="menu_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>保存接受数据</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>清除界面数据</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>串口</string>
   </property>
  </action>
  <action name="actionudp">
   <property name="text">
    <string>udp</string>
   </property>
  </action>
  <action name="actiontcp">
   <property name="text">
    <string>tcp</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>设置隐藏</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>设置打开</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>日志查看模式</string>
   </property>
  </action>
  <action name="action16">
   <property name="text">
    <string>16进制数据查看模式</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
